### ISSUE TITLE
Separate each module into separately built jar as well as the unified jar

### ISSUE THREAD

**Issue by richiksc** [DurhamAcademy/kyberlib#6]
Hi, we're thinking about doing AddressableLEDs on our robot and saw that you guys had already created an awesome library for it. Thanks! However, is there any way to get a jar of only the `lighting` module?

**Comment by trevnels** [comment]
Yep, that's something I'll consider. For now you should be able to just copy/paste the lighting-specific classes into your project until I get a separate jar up.

**Comment by richiksc** [comment]
@trevnels We use Java and not Kotlin so that makes it a little more difficult. We don't want to add the Kotlin runtime. IntelliJ IDEA does have a built in Kotlin to Java convert feature, however.
